docs/monthly-meeting/2023-09.md

```diff
@@ -0,0 +1,118 @@
+
+- [Adam] [Datamodel headings](https://github.com/python/cpython/pull/108146) for readability
+
+- [Adam] Work to [remove references to Distutils](https://github.com/python/cpython/issues/92584) in the documentation ahead of the 3.12.0 release
+
+
+## Discussion
+
+- [Hugo] python-docs-theme:
+  - Need a release of 2023.8 to fix some dark theme bugs: [python/python-docs-theme#152](https://github.com/python/python-docs-theme/issues/152)
+  - Have a PR ready to release using Trusted Publishers, need someone to set it up on PyPI: [python/python-docs-theme#148](https://github.com/python/python-docs-theme/pull/148)
+  - These are both waiting on people who can make releases. Adam: should we review the group of people who can make releases, and look to expand it? Potentially ties in to a [Python organisation](https://github.com/python/steering-council/issues/207) on PyPI.
+
+
+- [Ege/Hugo/CAM] Updates and discussion on Plausible hosting:
+  - PSF folks opposed to self-hosting
+  - Paid hosting is a *little* expensive
+  - We got a 15% discount from them, which would be \$1400-\$1500 yearly
+  - People brought up several motivating use cases already for this, including:
+    - Translation prioritization for languages/pages
+    - Automated translations
+    - Determine how many people use backported branches
+    - Determining if old pages are still needed/used
+    - Potentially determine common broken links, maybe even fragment ids?
+    - Prioritisation for documentation
+  - [Previous Core Development budget thread](https://discuss.python.org/t/23507)
+  - Next steps: CAM to write to Deb from the PSF with a summary of this discussion
+
+- [Maciej] Translations platform – trying to initiate discussion ([Discourse thread](https://discuss.python.org/t/docs-translation-platform/29940))
+  - Transifex has cut off shared memory (between projects) for open source tier, Rafael from Brazilian Portugese team implemented a workaround
+  - Weblate – based on Django, open license, used by PyPI
+  - The POC of Python docs translations/Weblate integration is functional, I'm working on improving it
+  - Python docs don't fit into free open source tier on Weblate – the initial proposition is 1500€/year (50% discount)
+  - There is an option to self-host a Weblate instance
+  - (Both platforms support using machine learning translation tools)
+
+- [Ege] New bootstrapper for starting new Python docs translations
+  - Issues with the [current tool](https://github.com/JulienPalard/python-docs-cookiecutter):
+    - Unmaintained / inactive
+    - Uses third-party modules
+  - Add a link from the [devguide](https://devguide.python.org/documentation/translating/#starting-a-new-translation) to this [repository](https://github.com/egeakman/python-docs-bootstrapper)
+  - [PEP 545](https://peps.python.org/pep-0545/#repository-for-po-files) mentions the per-language repositories for `.po` files
```

Comment:
I think I wrote this, the discussion briefly turned to having all translations managed online through a provider and I was looking for the citation that we should at least save the .mo files to Git. But probably irrelevant for these notes

Reply:
Removed